client_cli/src/main.rs

```diff
@@ -96,6 +102,8 @@ pub enum Subcommand {
     /// The subcommand related to event streaming
     #[clap(subcommand)]
     Events(events::Args),
+    /// The subcommand related to Wasm
+    Wasm(wasm::Args),
```

Comment:
note to self: update this in docs

https://github.com/hyperledger/iroha-2-docs/issues/216